locales/vi-VN/App.json

```diff
@@ -42,7 +42,7 @@
   "LOCKED": "KHOÁ",
   "Could not fetch data because the wallet is offline": "Không thể đẩy dữ liệu vì ví đang ngoại tuyến",
   "Unknown wallet name": "Tên ví không xác định",
-  "To remove this address from being the default address, you must set another one as main first.": "Bạn cần phải đặt một địa chỉ ví mặc định mới trước khi thay đổi địa chỉ ví mặc định này.",
+  "To remove this address from being the default address, you must set another one as default first.": "Bạn cần phải đặt một địa chỉ ví mặc định mới trước khi thay đổi địa chỉ ví mặc định này.",
```

Comment:
Now, the question comes, how do we handle this change...? @MaudSimon

Reply:
here you are 6c78910868c7ec701c2308786955e5196d879377